### DISCUSSION TITLE
QField doesn't show the captured photos in field

### DISCUSSION THREAD

**Discussion by stefanoch90** [opengisch/QField#3277] [Q&A]
I don't make to visualize photo to an object just took. QField takes photos and add the path and the name of the file but it seems QField doesn't save the file. QField displays an icon which announce it can't found the file. But the file is saved in the DCMI folder filed the project folder. 
When i created the project file in Qgis, i added in my attributes table, the "Photo" field. Finally, in Layer Properties-> Attributes Form, I set up the widget type to Attachment.

**Comment by ConnicksGIS** [comment]
Hi,

Unless I am missing something, I am also having this issue as described above. I create a project in QGIS, synch it to QField via QFieldCloud. Out in the field I add a feature and take a photo. I synch this back to QFieldCloud and the QGIS project. I can see the point I have recorded and the image file name but I cannot find the image saved anywhere in the cloud or within my QGIS project. 

Maybe I am missing something - any ideas?

![image](https://user-images.githubusercontent.com/122447390/211791369-f05612df-dcb1-4dbf-b0e4-c20fd83601fc.png)

**Comment by suricactus** [comment]
@stefanoch90 @ConnicksGIS can you please let us know what QField and device you are using?

**Comment by stefanoch90** [comment]
Hi Ivan, I had the same problem as you the first time I used QField. I solved it in the following way:1) I created a new project file in QGIS and saved the data inside a folder.2) I connected my smartphone to my notebook and dragged the QGIS project folder inside the QField folder in my smartphone.In this way I solved the problem of not displaying the image taken in the field.I hope this solution helps you as well.See you soon Ing. Stefano ChiappiniBuilding Engineer; PhD ResearcherUniversità Politecnica delle Marche - Dept. of Agricultural, Food and Environmental Sciences – D3AVia Brecce Bianche 10, 60131, Ancona, Italy
